color_refinement.py

```diff
@@ -103,6 +119,8 @@ def get_number_isomorphisms(g: Graph, h: Graph, count: bool) -> int:
     if len(g.edges) != len(h.edges):
```

Comment:
Isn't this also an preprocessing step in your definition?

Reply:
Note to self: is already in `check_degrees` and `check_graph_size`
So you could remove this two checks